<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicView</class>
 <widget class="QFrame" name="BasicView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetting">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCfg">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayoutRecv">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutRecv" stretch="1,0,0">
         <item>
          <widget class="QLabel" name="labelRecv">
           <property name="text">
            <string>Receive Buffer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRecvTextMode">
           <property name="text">
            <string>Hex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClearRecv">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayoutRecv">
         <item row="0" column="0">
          <widget class="QTextEdit" name="textRecv">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayoutSend">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutSend" stretch="1,0,0,0">
         <item>
          <widget class="QLabel" name="labelSend">
           <property name="text">
            <string>Send Buffer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendTextMode">
           <property name="text">
            <string>Hex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendData">
           <property name="text">
            <string>Send Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendFile">
           <property name="text">
            <string>Send File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayoutSend">
         <item row="0" column="0">
          <widget class="QTextEdit" name="textSend"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnSendData</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
